confidence/src/main/java/com/spotify/confidence/Confidence.kt

```diff
@@ -112,6 +146,43 @@ class Confidence internal constructor(
         eventSenderEngine.emit(eventName, message, getContext())
     }
 
+    private val networkExceptionHandler by lazy {
+        CoroutineExceptionHandler { _, _ ->
+            // network failed, provider is ready but with default/cache values
+            print("")
```

Comment:
let's signal this in some better way?

Reply:
ah we don't need this, forgot to remove.